test/utils/plugins.js

```diff
@@ -24,7 +24,41 @@ function installPlugin(installDir, PluginClass) {
   );
 }
 
+function installESMPlugin(installDir, PluginClass) {
```

Comment:
Instead of creating this util. Let's simply extend the [`programmatic/plugin`](https://github.com/serverless/serverless/tree/main/test/fixtures/programmatic/plugin) fixture with (success) ESM examples, and test against that with single `runServerless` run. 

For failure testing, we can add there also those plugin cases, but test them with different `runServerless` runs, requiring then via `configExt` option